optuna/visualization/_parallel_coordinate.py

```diff
@@ -97,14 +114,43 @@ def _get_parallel_coordinate_plot(
     layout = go.Layout(title="Parallel Coordinate Plot")
     reverse_scale = _is_reverse_scale(study, target)
 
+    info = _get_parallel_coordinate_info(study, params, target, target_name)
```

Comment:
For testability, maybe we should bring `reverse_scale` inside `_ParallelCoordinateInfo` and make `_get_parallel_coordinate_plot` function receive a single `_ParallelCoordinateInfo`. This would enable splitting the test, i.e., we will not need to create a `Study` object in testing `_get_parallel_coordinate_plot`.